src/components/Calculator.js

```diff
@@ -1,17 +1,30 @@
 import React, { Component } from 'react';
 import Button from './Button';
+import calculate from '../logic/calculate';
 
```

Comment:
- [ ] While I can see you have imported calculate.js from the logic directory but I cannot see operate.js being imported here. Please import it as we need it here to run them.

Reply:
Could you explain please? operate.js is used by calculate.js. What is it that I need to do with operate.js?

Reply:
I consulted with TSE leader and please ignore this requirement as it's already been imported. 
Thank you for the catch